### PR TITLE
Fix up -Wcast-qual warning (#193)

### PR DESCRIPTION
**Pull request by clalancette**
* Fix up -Wcast-qual warning

* Change to C++-style cast

Co-Authored-By: Chris Lalancette <clalancette@gmail.com>

This is the forward-port to ros2 branch of #193

### CODE REVIEW

**Comment by clalancette** [comment]
All right.  This passed PR testing, passed local testing (both build and test), and was actually tested in https://github.com/ros2/geometry2/pull/193#issuecomment-554352889 as well.  @mjcarroll This should be good to go; once you've approved, I'll merge and then release geometry2 for Eloquent again.

**Review by mjcarroll — APPROVED** [review]

**Pull request merged**
